src/components/Comments.jsx

```diff
@@ -0,0 +1,50 @@
+import { fetchCommentsByArticleID } from "../api";
+import { useEffect, useState } from "react";
+import { sqlDateFormatter } from "../utils.js";
+
+const Comments = ({ article_id, comments, setComments }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    if (typeof article_id !== "undefined" && article_id !== "") {
+      fetchCommentsByArticleID(article_id)
+        .then((comments) => {
+          setComments(comments);
+          setIsLoading(false);
+        })
+        .catch(() => {
+          setIsError(true);
+          setIsLoading(false);
+        });
+    }
+  }, [article_id, setComments]);
+
+  if (isError) return <p>Sorry, we are unable to load comments at the moment</p>;
+  if (isLoading) return <p>Loading comments...</p>;
+
+  return (
+    <section className="Comments">
+      <h3>Comments</h3>
+      <ul className="comments-flex-container">
+        {comments.length === 0
+          ? "No comments - be the first to post one!"
```

Comment:
Excellent consideration of user experience here, and great job framing the message as a call to action.